src/misc/get-file-info.ts

```diff
@@ -0,0 +1,201 @@
+import * as fs from 'fs';
+import * as crypto from 'crypto';
+import * as fileType from 'file-type';
+import isSvg from 'is-svg';
+const probeImageSize = require('probe-image-size');
```

Comment:
```suggestion
import * as probeImageSize from 'probe-image-size';
```

#3713 型定義いると思いますが、書きましょうか？

Reply:
書いてもいいです
あまり複雑じゃないし、なんか型定義入れるとテストコケた気がするので作ろうとしませんでした

Reply:
`TS_NODE_FILES=true`あたり付ければ型定義読んでくれる？？？

Reply:
なるほど。テストで型読んでもらえずコケるために型つけないのはテストする意味がないと思うので、なんらかの形で解決する必要はありますね。

Reply:
`TS_NODE_FILES=true`付けてテストすれば問題ないぽいので、ざっとd.ts追加しました。
(多分網羅はしてないけど)